### ISSUE TITLE
Dragable across X Axsis 

### ISSUE THREAD

**Issue by joshhornby** [wintvelt/elm-sortable-list#2]
Hi,

This is a great example, would be nice if we could also drag across the X axsis for lists that are horizontal.

**Comment by wintvelt** [comment]
Yeah, that might be a nice alternative/ extension. Must admit it has been a while since I work on the code (unfortunately). Sorry for the late reply.
But maybe I'll get an opportunity to do that. This simple example relies on a fixed height for each of the items in the vertical list (which is quite common). A simple example for horizontal version would require a fixed width for all items too, but then it would be relatively straightforward I guess.
